src/inmanta/ast/type.py

```diff
@@ -250,8 +250,11 @@ def as_python_type_string(self) -> "str | None":
     def has_custom_to_python(self) -> bool:
         return False
 
+    def type_string(self) -> str:
+        return "any"
```

Comment:
This is not right. This method must really only be implemented for types that can be expressed directly in the DSL (this one only exists on the plugin boundary).